Source/SelfService/Web/logging/logFilter/filterSelect.tsx

```diff
@@ -2,8 +2,35 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 import React from 'react';
+
+import { themeDark } from '../../theme/theme';
 import { Select, SelectProps } from '@mui/material';
 
+const styles = {
+    'padding': '5px 14px',
+    'borderRadius': '4px',
+    'transition': 'all .3s',
+    '&:before, &:hover:not(.Mui-disabled):before, &:after': {
+        border: 'none',
+    },
+    ':hover': {
+        'background': 'rgba(140, 154, 248, 0.08)',
+        '& svg': {
+            color: themeDark.palette.primary.main
```

Comment:
I'm glad you are using the theme color here. Instead of using it directly from the theme file, you should lean on the theme callback, so it always uses the correct color. This will also apply if we dynamically change themes (ex: Light/Dark)

I don't know the syntax off-hand, but something like:

```ts
    color: (theme) => theme.palette.primary.main
```